iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/stack.inl

```diff
@@ -45,8 +46,10 @@ inline bool stack<T, Capacity>::push(Targs&&... args) noexcept
         return false;
     }
 
-    // low level memory management with access to the topmost element on the untyped buffer;
-    // type safety is ensured by the template parameter
+    // AXIVION Next Construct CertC++-MEM54, CertC++-EXP36, AutosarC++19_03-A18.5.10,
```

Comment:
CertC++-MEM54: placement new is ok since the input pointer is aligned for the constructed type and there is sufficient space. This is ensured by design (definition of `element_t`) with the correction proposed.

AutosarC++19_03-A18.5.10: same, with the changes the elements should be correctly aligned and provide sufficient space.

Ideally static code analysis will not detect any violation after the change to `element_t`.

Reply:
Removed the suppressions since I can not reproduce the warnings.